.github/workflows/check-by-name.yml

```diff
@@ -91,8 +91,12 @@ jobs:
           git worktree add "$base" "$(git rev-parse HEAD^1)"
           echo "base=$base" >> "$GITHUB_ENV"
       - uses: cachix/install-nix-action@7ac1ec25491415c381d9b62f0657c7a028df52a7 # v24
-      - name: Fetching the tool
-        run: pkgs/test/nixpkgs-check-by-name/scripts/fetch-pinned-tool.sh result
+      - name: Fetching the pinned tool
+        run: |
+          # Get the direct /nix/store path from the pin to avoid having to evaluate Nixpkgs
+          toolPath=$(jq -r .ci-path pkgs/test/nixpkgs-check-by-name/scripts/pinned-tool.json)
```

Comment:
Should've tested this in a separate repository. This broke CI on master very briefly before I fixed it with https://github.com/NixOS/nixpkgs/pull/281412. Only two PRs were affected:
- https://github.com/NixOS/nixpkgs/pull/281621#issuecomment-1896641435
- https://github.com/NixOS/nixpkgs/pull/281622#issuecomment-1896643055